### ISSUE TITLE
Intentos

### ISSUE THREAD

**Issue by caroacostatovany** [ElenaVillano/sentiment_analysis_tweets#8]
- [x] - Balancear los datos de entrada:
          - [ ] - Todos que sean de la misma longitud de tokens
- [ ] - Encontrar el max de vocabulario
- [x] - Intentar con Bidirectional
- [x] - Intentar sigmoid o meter una capa más
- [x] - Intentar con from_logits=False
- [x] - Intentar con 22,000 datos balanceados en etiqueta
- [x] u -> you
- [x] Quitar user_mention y quitar URL
- [x] - Hacer padding al principio y al final y ver comparaciones
